spec/models/field_value_report_spec.rb

```diff
@@ -68,8 +68,7 @@
     end
 
     it 'returns "__MISSING__" for field without value' do
-      r = described_class.find('sourceResource_alternative_providedLabel',
-                               provider_id)
+      r = described_class.find('sourceResource_date_name', provider_id)
```

Comment:
Another case where we need a `Maybe` monad.

![image](https://cloud.githubusercontent.com/assets/1361776/17571096/ae3e8f20-5f03-11e6-9314-032beb717ee4.png)